README_maintainers.md

```diff
@@ -642,209 +624,261 @@ These commands for `-c` mean:
+2012-01-28T19:10:23Z [10671] INFO 0 tests failed
+2012-01-28T19:10:23Z [10671] INFO 1 millisecond elapsed
 ```
 
-Point your webbrowser to the IP of your `echo "http://${MESOS_IP}:8080"`.
+#### Running jsUnity Tests with arangod
 
-Wait until arangodb is healthy.
+In (emergency) console mode (`arangod --console`):
 
-Then click on `arangodb`.
+    require("jsunity").runTest("tests/js/server/aql/aql-escaping.js");
 
-On the following screen click on the first port next to the IP Address of your cluster.
+Filtering for one test case (in this case `testTokens`) in console mode:
 
-Deploying a locally changed version
------------------------------------
+    require("jsunity").runTest("tests/js/server/aql/aql-escaping.js", false, "testTokens");
 
-Create local docker images using the following repositories:
+Run a test via startup option:
 
- - https://github.com/arangodb/arangodb-docker
- - https://github.com/arangodb/arangodb-mesos-docker
- - https://github.com/arangodb/arangodb-mesos-framework
+    build/bin/arangod /tmp/dataUT --javascript.unit-tests="tests/js/server/aql/aql-escaping.js" --no-server
 
-Then adjust the docker images in the config (`arangodb3.json`) and redeploy it using the curl command above.
+    js/common/modules/loadtestrunner.js
 
---------------------------------------------------------------------------------
-Front-End (WebUI)
-=========
+#### Running jsUnity Tests with arangosh client
 
-To build the current frontend build please use the command: `make frontend`.
-The command `make frontend_clean` will remove all available node modules and
-start a clean installation.
+You can only run tests which are intended to be ran via arangosh:
 
---------------------------------------------------------------------------------
-NPM dependencies
-=======
+    require("jsunity").runTest("tests/js/client/shell/shell-client.js");
 
-To add new NPM dependencies switch into the `js/node` folder and install them
-with npm using the following options:
+### Mocha Tests
 
-`npm install [<@scope>/]<name> --global-style --save --save-exact`
+All tests with `-spec` in their names are using the [mochajs.org](https://mochajs.org) framework.
+To run those tests, e.g. in the arangosh, use this:
 
-or simply
+```js
+const runTest = require("@arangodb/mocha-runner");
+runTest(<filepath>, true)
+```
 
-`npm install [<@scope>/]<name> --global-style -s -E`
+### Debugging Tests
 
-The `save` and `save-exact` options are necessary to make sure the `package.json`
-file is updated correctly.
+This is quick introduction only.
 
-The `global-style` option prevents newer versions of npm from unrolling nested
-dependencies inside the `node_modules` folder. Omitting this option results in
-exposing *all* dependencies of *all* modules to ArangoDB users.
+Running a single rspec test:
 
-Finally add the module's licensing information to `LICENSES-OTHER-COMPONENTS.md`.
+   ./scripts/unittest http_server --test api-import-spec.rb
```

Comment:
line has to stand alone to make code-boxes work

Reply:
was actually missing an indention space